newsletters/2023-09.md

```diff
@@ -0,0 +1,69 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/FIX THIS LINK)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter September 2023
+
+TL;DR: In this newsletter we announce the OBF's upcoming public board meeting; provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 
+
+In longer form: 
+
+## OBF News
+
+### Upcoming public board meeting
+
+The OBF is having its next public board meeting on XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog [add link]. 
+
+### BOSC news
+
+The 24th annual Bioinformatics Open Source Conference (BOSC 2023) took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 
+
+In addition to 43 talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 
+
+You can [read the full report about the conference on the OBF blog](https://www.open-bio.org/2023/08/14/bosc-2023-report/).
```

Comment:
Add:

For those who like to plan ahead, BOSC 2024 will take place between July 12-16, 2024, as part of [ISMB 2024](https://www.iscb.org/ismb2024) in Montréal, Québec, Canada.